cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -192,9 +192,9 @@ MACRO(DEAL_II_ADD_TEST _category _test_name _comparison_file)
         ENDIF()
 
         IF(NOT "${TEST_TARGET_${_build}}" STREQUAL "")
-          SET(_target ${TEST_TARGET_${_build}})
+          SET(_target ${_category}.${TEST_TARGET_${_build}})
         ELSEIF(NOT "${TEST_TARGET}" STREQUAL "")
-          SET(_target ${TEST_TARGET})
+          SET(_target ${_category}.${TEST_TARGET})
```

Comment:
Do we need to append the error message in line 201 below as well or is it clear nonetheless?

Reply:
I would say `_target` is only function internal, while `${TEST_TARGET}` come from outside which is the motivation for the error message.